db/migrate/20241016045036_create_enrollments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEnrollments < ActiveRecord::Migration[7.2]
+  def change
+    create_table :enrollments do |t|
+      t.references :student, null: false, foreign_key: { to_table: :users }
```

Comment:
As far as I could see student has a belongs_to relation with enrolment. So why not use it like this in migration:
 
```suggestion
      t.belongs_to :student, foreign_key: true
```

Reply:
Committed the suggestion apu. Thank you.